src/couch_mrview/src/couch_mrview_debug.erl

```diff
@@ -48,3 +86,159 @@ view_signature(DbName, DDocName) ->
     {ok, DDoc} = couch_db:open_doc_int(Db, <<"_design/", DDocName/binary>>, []),
     {ok, IdxState} = couch_mrview_util:ddoc_to_mrst(DDocName, DDoc),
     couch_util:to_hex(IdxState#mrst.sig).
+
+first_call(Pid) ->
```

Comment:
The `first_call/1` and `find_by_first_call` are generic enough and have utility in other debugging activities. Please move them into `couch_debug.erl`. Don't forget to document them. Since they would become public operator facing API.

Reply:
I do not think they need to be written differently. We could move them to `couch_debug` and they could be used for other functions in different modules.

Reply:
But I will save that for another PR.

Reply:
I have removed `first_call`, `find_by_first_call`, and `get_indexer_pid` from the PR. They were initially only added for testing purposes.